docs/rules/no-unused-modules.md

```diff
@@ -0,0 +1,101 @@
+# import/no-unused-modules
+
+Reports:
+  - modules without any or
+  - individual exports not being used within other modules
+
+Note: dynamic imports are currently not supported.
+
+## Rule Details
+
+
+### Options
+
+This rule takes the following option:
+
+- `src`: an array with files/paths to be analyzed. It only for applies for unused exports. Defaults to `process.cwd()`, if not provided
+- `ignoreExports`: an array with files/paths for which unused exports will not be reported (e.g module entry points) 
+- `missingExports`: if `true`, files without any exports are reported  
+- `unusedExports`: if `true`, exports without any usage within other modules are reported.
```

Comment:
If both missingExports and unusedExports are missing or false, what gets reported?

If nothing, can we ensure it’s a schema error if neither of them are true?

Reply:
I'm lacking experience with the JSON schema. Do you have an idea or example on how to conditionally require one or the other field?

Reply:
Finally figured it out. We now get a schema error, if both are missing/false.